t/loc_tools.pl

```diff
@@ -589,7 +591,7 @@ ($)
     }
 
     die "Couldn't restore locale '$save_locale'"
-        unless setlocale(&POSIX::LC_CTYPE(), $save_locale);
+                            unless setlocale(&POSIX::LC_CTYPE(), $save_locale);
```

Comment:
I don't understand why you are increasing the indent so much in the line above.

Reply:
It is a style used in some spots of the core.   If a line is broken so as to make it fit in the recommended line width, a hanging indent is created to make the second line end close to that max width.  It makes that line more obviously a continuation, and the eye doesn't have to move so far to the left.  I use it sometimes when it seems like the right thing.  I don't know of any discussions about this style; I certainly didn't originate it.